src/openforms/prefill/sources/user_defined.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from rest_framework.exceptions import ValidationError
+
+from openforms.submissions.models import Submission
+from openforms.submissions.models.submission_value_variable import (
+    SubmissionValueVariable,
+)
+from openforms.typing import JSONEncodable
+
+from ..registry import Registry
+
+logger = logging.getLogger(__name__)
+
+
+def fetch_prefill_values(
```

Comment:
does this still work if I have a user defined variable configured with plugin + attribute rather than plugin + options?